src/main/java/appeng/integration/modules/crafttweaker/CTModule.java

```diff
@@ -0,0 +1,47 @@
+
+package appeng.integration.modules.crafttweaker;
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+import net.minecraft.item.ItemStack;
+
+import crafttweaker.CraftTweakerAPI;
+import crafttweaker.IAction;
+import crafttweaker.api.item.IItemStack;
+
+import appeng.integration.abstraction.ICraftTweaker;
+
+
+public class CTModule implements ICraftTweaker
+{
+	static List<IAction> ADDITIONS = new ArrayList<>();
+	static List<IAction> REMOVALS = new ArrayList<>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "REMOVALS" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS3008)